better-font-awesome.php

```diff
@@ -513,18 +543,67 @@ public function text_callback( $args ) {
      * @return  string  Usage text.
      */
     public function get_usage_text() {
```

Comment:
`plugin_dir_path( __FILE__ )` or `__DIR__`?

__DIR__ is a shortcut for dirname( __FILE__ ) but isn't PHP 5.2 compatable.

Internally `plugin_dir_path` just does `trailingslashit( dirname( __FILE__ ) );`

I'd probably use `plugin_dir_path` in a plugin - but often I do just use __DIR__.